Wilson.sln

```diff
@@ -105,7 +105,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{BC99A01F
 		build\dependenciesTest.props = build\dependenciesTest.props
 		build\targets.props = build\targets.props
 		build\targetsTest.props = build\targetsTest.props
-		build\version.props = build\version.props
```

Comment:
why remove from the sln?

Reply:
It's deleted

Reply:
I'm sorry, I don't see version.props as deleted

Reply:
@jennyf19, I had readded during the WE it as it's also used by testscommons. But it's now different.
I've rea-added it to the solution too (right now)